### PR TITLE
Node e2e junit test artifacts

### LINKED ISSUE #25966: Kubelet e2e tests are not compatible with merge queue logic
Kubelet e2e tests are run manually using go command. As a result jenkins treats test failures as "red" ones not "yellow". AFAIU this means that we will block with the first flake, not two in the row as for other suites.

@kubernetes/sig-node @lavalamp @wojtek-t

### PR DESCRIPTION
**Pull request by pwittrock**
[Analytics]()
- Add junit test reported
- Write etcd.log, kubelet.log and kube-apiserver.log to files instead of stdout
- Scp artifacts to the jenkins WORKSPACE

Fixes #25966

### CODE REVIEW

File: test/e2e_node/jenkins/copy-e2e-image.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```
Comment (vishh) [inline test/e2e_node/jenkins/copy-e2e-image.sh]: Thanks for adding this.

Comment (vishh) [inline test/e2e_node/jenkins/copy-e2e-image.sh]: Nest step: Run this automatically if images don't exist.

**Comment by vishh** [comment]
LGTM

File: test/e2e_node/jenkins/copy-e2e-image.sh (hunk 1 above)
Comment (pwittrock) [inline test/e2e_node/jenkins/copy-e2e-image.sh]: That is a good idea.

**Comment by pwittrock** [comment]
kubernetes/test-infra#38

**Comment by pwittrock** [comment]
@k8s-bot e2e test this issue: #26138

**Comment by pwittrock** [comment]
PTAL.  All the files seem to be in the right place now.  Do you have any idea why the kubelet.log has non-printable characters in it?

**Comment by pwittrock** [comment]
@k8s-bot e2e test this issue: #26139

**Comment by pwittrock** [comment]
@k8s-bot e2e test this issue: #26187

**Comment by pwittrock** [comment]
Removed LGTM so I can squash the fixups, will re-apply

**Comment by pwittrock** [comment]
Squashed the commits.

**Comment by pwittrock** [comment]
Added P2 label since this is needed by our test infrastructure

**Comment by pwittrock** [comment]
Bumping the priority since we need to generate test results for assigning test ownership.

**Pull request merged**